app/src/server/scripts/connectUsageWithInvoices.ts

```diff
@@ -0,0 +1,24 @@
+import dayjs from "dayjs";
+import { kysely, prisma } from "../db";
+
+const startOfFeb2024 = dayjs("2024-02-01").startOf("month").toDate();
+const endOfFeb2024 = dayjs("2024-02-01").endOf("month").toDate();
+
+const invoices = await prisma.invoice.findMany({
+  where: {
+    billingPeriod: "Feb 2024",
+  },
+});
+
+for (const invoice of invoices) {
+  await kysely
+    .updateTable("UsageLog")
```

Comment:
We should only update usage where it's currently null. We don't want to ever pull usage away from an existing invoice.

Reply:
I think `.where("invoiceId", "is", null)` already does this?

Reply:
Apologies, missed that!